plugins/scenesPageRememberStates/scenePageRememberStates.js

```diff
@@ -0,0 +1,101 @@
+(async function waitForStash() {
+  while (!window.stash) {
```

Comment:
window.stash is never exposed and being a requirement should ensure it's loaded beforehand